client/src/screens/TallyScreen.tsx

```diff
@@ -392,6 +392,9 @@ const TallyScreen = () => {
             <HorizontalRule />
             <h2>Pre-Election Features</h2>
             <p>
+              <LinkButton to={routerPaths.printTestDecks}>
+                Print Test Decks
+              </LinkButton>
```

Comment:
Need to force a space here such that the buttons don't touch in the UI.
```suggestion
              </LinkButton>{' '}
```